static_src/components/app_list.jsx

```diff
@@ -65,7 +69,11 @@ export default class AppList extends React.Component {
     );
 
     if (this.state.empty) {
-      content = <h4 className="test-none_message">No apps</h4>;
+      content = (
+        <EntityEmpty callout="You have no apps in this space">
+          <InfoAppCreate space={ this.state.currentSpace } org={ this.state.currentOrg } />
```

Comment:
Yeah, I think you still need to pass in a user to `props` for it to do the check.